### ISSUE TITLE
Cannot craft wood planks + random outputs from crafting grid with nothing in it.

### ISSUE THREAD

**Issue by WillTereika** [MSandro/NeoForge-Foundry-StoneBlock-Edition#1]
I cannot craft wood planks, All wood, and in fact MOST items in the crafting grid, want to turn into sawdust.  Many other random items also randomly appear in the crafting output, like titanium plates and other blocks.  Something VERY weird is going on with the crafting system in the modpack.

**Comment by DiggityMan420** [comment]
happen to me too I had to load it a few times but it works now and hasn't happen again yet and I didn't even change anything just restart Minecraft.

**Comment by MSandro** [comment]
Yes it is a known bug, I've already spent a lot of time trying to fix it, but without success. I haven't been able to find out which mod is causing it yet. It just happens very randomly.
